app/scripts/views/sign_up.js

```diff
@@ -326,37 +328,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
             self.logScreenEvent('preverified.success');
           }
           self.logScreenEvent('success');
-
-          if (self.relier.accountNeedsPermissions(account)) {
-            self.navigate('signup_permissions', {
-              data: {
-                account: account
-              }
-            });
-
-            return;
```

Comment:
related to https://github.com/mozilla/fxa-content-server/pull/3121/files#r41161346 

The account is not created until permissions are accepted